cf/test/test_Data.py

```diff
@@ -114,6 +114,18 @@ def setUp(self):
                 "ignore", category=RuntimeWarning, message=expected_warning
             )
 
+    def test_Data__init__basic(self):
+        """Test basic `__init__` cases for Data."""
+        # Most __init__ parameters are covered by the various other tests,
+        # so this is mainly to check trivial cases and especially the edge
+        # case of 'default' Data i.e. if no parameters are specified.
+        cf.Data(0, "s")
+        cf.Data(array=np.arange(5))
+        cf.Data(source=self.filename)
```

Comment:
An interesting choice of `source` (i.e. a string), but it certainly proves the test :)

Reply:
Thanks for clarifying David. That all makes sense.

> ... so we should perhaps consider improving this docstring in another PR ...

Good idea. I'll raise an Issue linking to this thread, update the line in question to specify something normal and sensible for the `source` parameter, and merge the PR.